identity/identity.go

```diff
@@ -193,3 +193,36 @@ func (i Identity) GetID() uuid.UUID {
 func (i Identity) GetNID() uuid.UUID {
 	return i.NID
 }
+
+func (i Identity) MarshalJSON() ([]byte, error) {
+	type localIdentity Identity
+	i.Credentials = nil
```

Comment:
Does this have side effects? Meaning that using `MarshalJSON` will set `Credentials` to `nil`? I'm always not sure with pass by ref / pass by value...

If it has side effects you could also do use `sjson.Delete()` to remove the key

Reply:
I first had a test to prove it doesnt :-). "i" is passed by value here.